libbpf-rs/tests/bpf_object_regen.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
```

Comment:
When I try running this script locally on the latest master, I'm seeing 
```
[~/local/libbpf-rs/libbpf-rs/tests] ./bpf_object_regen.sh
bin/src/percpu_map.bpf.c:3:10: fatal error: 'bpf/bpf_helpers.h' file not found
#include <bpf/bpf_helpers.h>
         ^~~~~~~~~~~~~~~~~~~
1 error generated.
bin/src/ringbuf.bpf.c:4:10: fatal error: 'bpf/bpf_helpers.h' file not found
#include <bpf/bpf_helpers.h>
         ^~~~~~~~~~~~~~~~~~~
1 error generated.
bin/src/runqslower.bpf.c:4:10: fatal error: 'bpf/bpf_helpers.h' file not found
#include <bpf/bpf_helpers.h>
         ^~~~~~~~~~~~~~~~~~~
1 error generated.
bin/src/taskiter.bpf.c:3:10: fatal error: 'bpf/bpf_helpers.h' file not found
#include <bpf/bpf_helpers.h>
         ^~~~~~~~~~~~~~~~~~~
1 error generated.
```

Does it compile cleanly for you?

Reply:
Yes, but I have a full libbpf as provided by my distribution installed.  Meaning that I have /usr/include/bpf/bpf_helpers.h on my machine.

libbpf-sys provides the bpf helpers for a normal compilation process of libbpf-rs and libbpf-cargo, but this is a raw compile to generate the objects without the libbpf-cargo assistance EG outside of the project.

I'm not sure if there is a non-complex way of grabbing the version of the headers as provided by the github-repo version of libbpf that libbpf-sys uses.  I can look into this if it's demanded, but it would create a very complicated script, and the intention was to be simple to read/understand, as it should be used very rarely.

I would prefer to update the test's readme to tell users to grab the appropriate packages from their distribution.

Reply:
Awesome, do you mid updating the test's readme to include this info? thanks!

Reply:
Done, all distributions described (unless explicitly marked untested) have been tested with the procedures given. (pfeww I need a better method of testing multiple distros)